indexer/api/api.go

```diff
@@ -16,17 +16,15 @@ type PaginationResponse struct {
 	HasNextPage bool        `json:"hasNextPage"`
 }
 
-func (a *Api) DepositsHandler(w http.ResponseWriter, r *http.Request) {
-	bv := a.bridgeView
-
+func (a *Api) L1DepositsHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
It is redundant since all Deposits are L1 but I bet this changes in future if we start supporting L2 to L2 bridging so I like this change

Reply:
yea that's what I was thinking